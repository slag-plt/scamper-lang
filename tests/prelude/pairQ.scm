(pair? (cons #t #f))
(pair? (+ 1 2))
(pair? "bye")
(pair? (cons "a" "b"))
(pair? (/ 100 1))