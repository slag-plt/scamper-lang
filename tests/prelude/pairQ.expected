#t
#f
#f
#t
#f
